src/scripts/main.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+
+function sortData(PositionCell, PositionName) {
+  const rowsArray = Array.from(tableBody.rows);
+
+  switch (PositionName) {
+    case 'Age':
+      rowsArray.sort((rowA, rowB) => {
+        return rowA.cells[PositionCell].innerHTML
+         - rowB.cells[PositionCell].innerHTML;
+      });
+      break;
+
+    case 'Name':
+      rowsArray.sort((rowA, rowB) => {
+        return rowA.cells[PositionCell].innerHTML.localeCompare(
+          rowB.cells[PositionCell].innerHTML);
+      });
+      break;
+
+    case 'Position':
+      rowsArray.sort((rowA, rowB) => {
+        return rowA.cells[PositionCell].innerHTML.localeCompare(
+          rowB.cells[PositionCell].innerHTML);
+      });
+      break;
```

Comment:
combine these checks.  for example, you don't need to do `localeCompare` twice, just combine `name` and `position` cases. make it simpler